defs/scenarios/storage/ceph/ceph-osd/bugs.yaml

```diff
@@ -26,6 +26,18 @@ checks:
       - varops: [[$bluestore_cache_onode], [contains, 7]]
       - varops: [[$bluestore_cache_onode], [contains, 8]]
       - varops: [[$bluestore_cache_onode], [contains, 9]]
+  has_53729:
+    apt:
+      ceph-osd:
+        - min: 15.0.0
+          max: 15.2.17
+        - min: 16.0.0
+          max: 16.2.9
+        - min: 17.0.0
+          max: 17.2.0
+  has_53729_osd_log:
```

Comment:
The original bug report mentioned it..but I don't think "log_to_monitors" log entry has any significance. So this can be removed.